src/client/dcp_client/app.py

```diff
@@ -6,6 +6,10 @@
 from dcp_client.utils import utils
 from dcp_client.utils import settings
 
+#changed from here
+from skimage.feature import canny, peak_local_max
+import numpy as np
+import cv2
```

Comment:
please see [a8573f5](https://github.com/HelmholtzAI-Consultants-Munich/data-centric-platform/pull/49/commits/a8573f50c920e93f37d45e5b605156c6202f40e8). I removed the opencv dependency and replaced it with skim age functions